packages/lmi/tests/test_llms.py

```diff
@@ -391,6 +391,90 @@ def test_pickling(self, tmp_path: pathlib.Path) -> None:
         assert llm.config == rehydrated_llm.config
         assert llm.router.deployment_names == rehydrated_llm.router.deployment_names
 
+    @pytest.mark.asyncio
+    async def test_acompletion_iter_logprobs_edge_cases(self) -> None:
+        """Test that acompletion_iter handles various logprobs edge cases gracefully."""
+        model = LiteLLMModel(name=CommonLLMNames.OPENAI_TEST.value)
+        messages = [Message(content="Say hello", role="user")]
+
+        def _build_mock_completion(
+            model: str = "test-model",
+            logprobs: Any = None,
+            delta_content: str = "",
+            delta_reasoning_content: str = "hmmm",
+            delta_role: str = "assistant",
+            usage: Any = None,
+        ) -> Mock:
+            mock_completion = Mock()
+            mock_completion.model = model
+            mock_completion.choices = [Mock()]
+            mock_completion.choices[0].logprobs = logprobs
+            mock_completion.choices[0].delta = Mock()
+            mock_completion.choices[0].delta.content = delta_content
+            mock_completion.choices[0].delta.reasoning_content = delta_reasoning_content
+            mock_completion.choices[0].delta.role = delta_role
+            if usage:
+                mock_completion.usage = usage
+            return mock_completion
+
+        # Mock the router to return different logprobs scenarios
+        with patch.object(model, "_router") as mock_router:
+            # Mock completion with None logprobs
+            mock_completion_none = _build_mock_completion(delta_content="Hello")
+
+            # Mock completion with logprobs but no content
+            mock_logprobs_no_content = Mock()
+            mock_logprobs_no_content.content = None
+            mock_completion_no_content = _build_mock_completion(
+                logprobs=mock_logprobs_no_content, delta_content=" world"
+            )
+
+            # Mock completion with empty content list
+            mock_logprobs_empty = Mock()
+            mock_logprobs_empty.content = []
```

Comment:
```suggestion
            mock_logprobs_empty = Mock(content=[])
```

Let's have these be one liners, easier to read imo

Reply:
Absolutely